### PR TITLE
Rebranded the application as "Symfony Demo"

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #85

### CODE REVIEW

File: app/Resources/translations/messages.ru.xliff [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
             <trans-unit id="title.homepage">
                 <source>title.homepage</source>
-                <target><![CDATA[Добро пожаловать в <strong>Symfony Демо Приложение</strong>]]></target>
+                <target><![CDATA[Добро пожаловать в <strong>Symfony Demo</strong> Приложение]]></target>
```
Comment (bocharsky-bw) [inline app/Resources/translations/messages.ru.xliff]: Lowercase should be like `приложение`.

File: app/Resources/translations/messages.uk.xliff [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
             <trans-unit id="title.homepage">
                 <source>title.homepage</source>
-                <target><![CDATA[Ласкаво просимо до <strong>Symfony Демо Додатку</strong>]]></target>
+                <target><![CDATA[Ласкаво просимо до <strong>Symfony Demo</strong> Додатку]]></target>
```
Comment (bocharsky-bw) [inline app/Resources/translations/messages.uk.xliff]: Lowercase should be like `додатку`.

**Comment by bocharsky-bw** [comment]
@javiereguiluz Could you apply my notes about lowercase words?

**Comment by javiereguiluz** [comment]
@bocharsky-bw done! Sorry for having changed this blindly and thanks for noticing me.

File: app/Resources/translations/messages.de.xliff [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
             <trans-unit id="title.homepage">
                 <source>title.homepage</source>
-                <target><![CDATA[Willkommen zur <strong>Symfony Demo Application</strong>]]></target>
+                <target><![CDATA[Willkommen zur <strong>Symfony Demo</strong> applikation]]></target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xliff]: Applikation

Comment (xabbuh) [inline app/Resources/translations/messages.de.xliff]: Though I think most Germans would use "Anwendung" or "Application".

Comment (javiereguiluz) [inline app/Resources/translations/messages.de.xliff]: Thanks. I've changed it by `Anwendung` because it's the only one that looks like real German :)

Comment (xabbuh) [inline app/Resources/translations/messages.de.xliff]: haha :+1:

**Pull request closed without merging**